include/onnxruntime/core/session/onnxruntime_cxx_api.h

```diff
@@ -963,8 +969,11 @@ struct SessionOptions : detail::SessionOptionsImpl<OrtSessionOptions> {
  *
  */
 struct ModelMetadata : detail::Base<OrtModelMetadata> {
-  explicit ModelMetadata(std::nullptr_t) {}                                   ///< Create an empty ModelMetadata object, must be assigned a valid one to be used
-  explicit ModelMetadata(OrtModelMetadata* p) : Base<OrtModelMetadata>{p} {}  ///< Used for interop with the C API
+  using Base = detail::Base<OrtModelMetadata>;
+  using Base::Base;
+
+  explicit ModelMetadata(std::nullptr_t) {}                                           ///< Create an empty ModelMetadata object, must be assigned a valid one to be used
+  explicit ModelMetadata(OrtModelMetadata* p) : detail::Base<OrtModelMetadata>{p} {}  ///< Used for interop with the C API
```

Comment:
nit: Is this needed since it is inherited from Base?

Reply:
Addressed